python/social_bot/envs/play_ground.py

```diff
@@ -177,7 +177,7 @@ def __init__(self,
             high=1.0,
             shape=[len(self._agent_joints)],
             dtype=np.float32)
-        if self._with_language:
+        if False and self._with_language:
```

Comment:
Why is here a "False" ?

Reply:
Why?

Reply:
during training, when computing logProb, it complains TypeError: Value passed to parameter 'x' has DataType uint32 not in list of allowed values...

Maybe somewhere we or TF is translating it into uint32.  I haven't got a chance to look deeper into it yet, and just picked a simpler fix to make our agent mute..

Reply:
We can set with_language to False through gin file to mute the agent.

Reply:
There are two fields for action, one is `control`, the other is `sentence`. The current ActionDistributionNetwork probably cannot handle it. Perhaps we should add a config like `with_agent_language`